tox.ini

```diff
@@ -1,9 +1,9 @@
 [tox]
 envlist =
     py{3.10,3.11,3.12}-django{5.0}-psycopg{2,3}
-    py{3.8,3.9,3.10,3.11,3.12}-django{4.2}-psycopg{2,3}
-    py{3.8,3.9,3.10,3.11}-django{4.0,4.1}
-    py{3.6,3.7,3.8,3.9,3.10}-django{3.2}
+    py{3.9,3.10,3.11,3.12}-django{4.2}-psycopg{2,3}
+    py{3.9,3.10,3.11}-django{4.0,4.1}
+    py{3.9,3.10}-django{3.2}
```

Comment:
can we:

- remove django 3.2 with python 3.6-3.7
- remove django 4.0 and django 4.1
- keep django 4.2 and keep python 3.8, ie. postpone python 3.8 removal with django 4.2 removal or when it will became blocker